### PR TITLE
Add ids.Struct_definition 

### PR DESCRIPTION
**Pull request by pefontana**
Update cairo-rs commit
Fix fn CairoRunner.get_output
Add Ids.Struct.SIZE to PyIds
Fix dict_read.cairo and dict_update.cairo test

### CODE REVIEW

**Review by Juan-M-V — APPROVED** [review]

File: src/ids.rs [hunk 1]
```diff
@@ -41,6 +45,30 @@ impl PyIds {
         if let Some(constant) = self.constants.get(name) {
             return Ok(constant.to_object(py));
         }
+
+        // Support for for ids.{Struct Definition} information
+        // Example: ids.DictAccess
+        let mut types_set = HashSet::new();
+        for key in self.struct_types.keys() {
+            types_set.insert(key.split('.').last());
```
Comment (Oppen) [inline src/ids.rs:53]:
```suggestion
            types_set.insert(key.rsplit('.').next().ok_or(...)?);
```

File: src/ids.rs [hunk 2]
```diff
@@ -41,6 +45,30 @@ impl PyIds {
...
+            let mut structs_size = HashMap::new();
+
+            for (key, v) in self.struct_types.iter() {
+                let max_member = v.values().max_by(|x, y| x.offset.cmp(&y.offset));
+
+                let max_offset = match max_member {
+                    Some(member) => member.offset + 1,
+                    _ => 0,
+                };
+                structs_size.insert(key.split('.').last(), max_offset);
```
Comment (Oppen) [inline src/ids.rs:65]:
```suggestion
                structs_size.insert(key.rsplit('.').next().ok_or(...)?, max_offset);
```

File: src/ids.rs (hunk 1 above)
Comment (Oppen) [reply src/ids.rs:53]: Please add a `TODO` comment, we should eventually move this logic to the program loading step so this one is just merging the `HashMap`s.

**Review by Oppen — APPROVED** [review]

**Comment by Oppen** [comment]
Now that I see the Cairo program more closely, I think it's the `import` clause that causes the type to be visible by plain name. Is there any way in the program to check the imports? It's only the imported variables and types that should be visible with those names so we can avoid collisions.

**Comment by pefontana** [comment]
> Now that I see the Cairo program more closely, I think it's the `import` clause that causes the type to be visible by plain name. Is there any way in the program to check the imports? It's only the imported variables and types that should be visible with those names so we can avoid collisions.

Yes, I make a test for checking this [cairo_programs/ids_size.cairo](https://github.com/lambdaclass/cairo-rs-py/pull/87/files#diff-63fae9c9a2fb0b436997d7463b6f0afe8c032bfa6dfc5e11fc94c36e1203e5e1)
You can check the SIZE of the imported structs or the ones that are defined on top.
If you defined a struct in main the ids.SIZE doesn't work in the PythonVM (In our implementation works OK)

File: src/ids.rs (hunk 1 above)
Comment (pefontana) [reply src/ids.rs]: I created an issue
https://github.com/lambdaclass/cairo-rs/issues/532

File: src/ids.rs (hunk 2 above)
Comment (pefontana) [reply src/ids.rs]: Done!

**Pull request merged**